keyboards/bastardkb/dilemma/4x6_4/info.json

```
@@ -0,0 +1,267 @@
+{
+    "keyboard_name": "Dilemma Max",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1837"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "matrix_pins": {
+        "cols": ["GP14", "GP8", "GP9", "GP7", "GP6", "GP28"],
+        "rows": ["GP15", "GP4", "GP5", "GP27", "GP26"]
+    },
+    "diode_direction": "ROW2COL",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "GP1",
+        "bootmagic": {
+            "matrix": [5, 0]
+        }
+    },
+    "encoder": {
+        "enabled" : true,
+        "rotary" : [
+            {"pin_a": "GP25", "pin_b": "GP24"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "caps_word": {
+        "enabled": true
+    },
```

Mostly a nit, this is easier to read/more compact: 
```suggestion
        "rgb_matrix": true,
        "caps_word": true
    },
```